docs/source/03_HED_formats.md

```diff
@@ -126,10 +128,13 @@ The location of the node element within the section specifies its relationship t
 
 A node element specifies a name,
 node attributes, and an informative description of the tag term's meaning.
-A node name may only contain alphanumeric characters, hyphen, and underscore.
-An exception to this is the `#` character which is used to represent a placeholder
-for a value to be provided during annotation.
-See [**SCHEMA_CHARACTER_INVALID**](./Appendix_B.md#schema_character_invalid) and
+A node name may only contain valid `name` characters 

```

Comment:
I just want to confirm this is an intentional change.

Currently allowed tag characters are alphanumeric and "-" only.  I don't foresee any implementation issues with changing this, just confirming the intent to add "." and "_".

Reply:
Yes I think we should allow.  I added the period because often qualified variables are specified as xxx.yyy.  I could imagine this being used and thought it would be reasonable as long as we don't run into some implementation problems.